tests/system/providers/google/cloud/bigquery/example_bigquery_queries.py

```diff
@@ -235,11 +236,21 @@
             trigger_rule=TriggerRule.ALL_DONE,
         )
 
+        openlineage_test = OpenlineageTestOperator(
```

Comment:
I believe it's advantageous to have OL system tests as an addition to regular ones, because they are just an addition of one operator, and actively use rest of the dag.

Those aren't only "OpenLineage system tests" in vacuum, but test OL integration of this particular provider.

Having separate dags for system tests would result in fragmentation - we'd have to maintain the same dag, or at least subset of it, with just the addition of a single `OpenLineageTestOperator`. 

However, I believe option to disable OpenLineage and make this special operator always pass is a very valid need, and will add such option.